specs/phase1/shard-fork-choice.md

```diff
@@ -59,7 +59,9 @@ def get_shard_latest_attesting_balance(store: Store, shard_store: ShardStore, ro
     return Gwei(sum(
         state.validators[i].effective_balance for i in active_indices
         if (
-            i in store.latest_messages and get_shard_ancestor(
+            i in store.latest_messages and
```

Comment:
In the rest of the repo, we've used the operation on the start of the next line (e.g. `and`) rather than the end of the current line.

Pretty sure this is a recent recommendation in PEP but can't currently find the standard